manifests/bosh-manifest/deployments/010-bosh-template.yml

```diff
@@ -9,7 +9,11 @@ meta:
     recursor: 8.8.8.8
 
   default_agent:
-    mbus: (( concat "nats://nats:" secrets.bosh_nats_password "@" meta.bosh_service_ip ":4222" ))
+    mbus: (( concat "nats://nats:" secrets.bosh_nats_password "@" terraform_outputs.bosh_fqdn ":4222" ))
+
+  director_instances:
```

Comment:
You mean director_instances? I'm not sure. It might not be used but...frankly speaking, this manifest needs a proper cleaning in the future.  I wanted to do a boy scout rule but that would ended up doing serious changes and I though it's not in the scope of this spike/story.